Add standard issue templates

wtrocki opened the pull request:
# Motivation

Seed collaboration by posting information about where discussion can be made and how to report issues.
Documents are mostly inspired by existing prisma/prisma .github folder that was updated recently.
I was thinking to add that early to somehow enforce good standards and collaboration for contributors. 
Any suggestions welcomed

# Tasks

- [x] Add bug fix
- [x] Add feature info
- [x] Add start page

Weakky commented:
Hey @wtrocki,

What are the purpose `bug_old.md` and `feature_request_old.md` ?

Thanks 👍

wtrocki commented:
@Weakky They were just different versions that were opened when users chose to do old issue 
<img width="1217" alt="screen shot 2019-01-13 at 11 46 24 am" src="https://user-images.githubusercontent.com/981838/51084875-34e48400-1729-11e9-826a-02d474f57078.png">

They were pointless duplicates I removed them. Also added and adjusted contributing file and Pull Request template. 

After merging this PR community status should look like this: 

<img width="1147" alt="screen shot 2019-01-13 at 11 50 29 am" src="https://user-images.githubusercontent.com/981838/51084889-7e34d380-1729-11e9-8a23-081c34418026.png">

I haven't added  code of conduct as this could make this PR more controversial. 
We can copy code of conduct from prisma repository:
https://github.com/prisma/prisma/blob/master/CODE_OF_CONDUCT.md 
Please let me know if you are happy with that. After that community status will be all green.

wtrocki commented on .github/ISSUE_TEMPLATE/documentation.md:
```diff
@@ -0,0 +1,7 @@
+---
+name: Documentation
+about: 'Please provide your feedback here instead: https://github.com/prisma/prisma-content-feedback/issues/new?template=documentation-feedback.md.'
+
```
I weren't sure if this will be part of the current prisma or it will have separate docs. 
I would say until documentation is not seeded somewhere else entire document can be removed ?

Weakky commented:
Thanks a lot for the PR 🙏
